ultravox/data/configs/commonvoice.py

```diff
@@ -0,0 +1,337 @@
+from ultravox.data import types
+
+CV_BASE_CONFIG = types.DatasetConfig(
+    name="commonvoice",
+    path="fixie-ai/common_voice_17_0",
+    transcript_template="{{sentence}}",
+    assistant_template="{{sentence}}",
+)
+
+CV_EN_CONFIG = types.DatasetConfig(
+    name="commonvoice-en",
+    base="commonvoice",
+    subset="en",
+    splits=[
+        types.DatasetSplitConfig(name="train", num_samples=1_101_170),
+        types.DatasetSplitConfig(name="validation", num_samples=16_393),
+    ],
+    transcript_template="{{text_proc.format_asr_text(sentence)}}",
+    assistant_template="{{text_proc.format_asr_text(sentence)}}",
+)
+
+CV_AR_CONFIG = types.DatasetConfig(
+    name="commonvoice-ar",
+    base="commonvoice",
+    subset="ar",
+    splits=[
+        types.DatasetSplitConfig(name="train", num_samples=28_369),
+        types.DatasetSplitConfig(name="validation", num_samples=10_470),
```

Comment:
So for example, here we should get used to writing types.DatasetSplitConfig(name="validation", num_samples=10_470, split_type="validation")